Tests/DataPipeline/Support/TestUtilities.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+@testable import Segment
+import XCTest
+
+// MARK: - Helper Plugins
+
+class OutputReaderPlugin: Plugin {
+    let type: PluginType
+    var analytics: Analytics?
+
+    var events = [RawEvent]()
+    var lastEvent: RawEvent?
+
+    init() {
+        self.type = .after
+    }
+
+    func execute<T>(event: T?) -> T? where T: RawEvent {
+        lastEvent = event
+        var eventSummary = ""
+
+        if let event = event {
+            eventSummary += " Event type: \(event.type ?? "nil")"
+            events.append(event)
+            if let event = event as? TrackEvent {
```

Comment:
We just want to print events of type `TrackEvent`?

Reply:
Just printed event name, which is available in `TrackEvent` only. For rest of the events e.g. `identify`, `type` is generally helpful.